packages/core/src/sass/app/modules/notifications-dialog.scss

```diff
@@ -156,6 +156,9 @@
             align-items: center;
             justify-content: center;
         }
+        &__content {
+            text-align: center;
+        }
```

Comment:
cant we pass this style to text component using align='center'?

Reply:
I've tried to pass align='center' to the text for this but it doesn't work, have to set it in scss only the text will be aligned to center